spec/30-processing-model.md

```diff
@@ -0,0 +1,64 @@
+# Processing Model
+
+This section defines the behaviour of a tracing system when a new request is received by a component. As stated above tracing systems can decide for each individual request whether they want to _forward_ a trace context or _participate_ in a trace context. The processing model for both scenarios will be described individualy and adds a seperate processing model when a tracing system is the first system to start a trace. 
```

Comment:
Terminology-wise should it be larger than just "tracing system"? Let's call it "tracing system or platform" to be more inclusive of proxy servers and such

Reply:
Ehhh ... I'm not sure if this distinction matters

Reply:
Platform is a very generic term I am not sure how this adds value

Reply:
We should at some point go through the spec and harmonize the terminology. Maybe move all of the explanations into some glossary.
I agree that a clear terminology is key but this should for now not keep us from merging.
Do you agree?

Reply:
Decided to postpone in cleanup phase later as difference is not substantial